<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mon Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="videoLabel">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="text">
       <string>Flux Vidéo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="messageLineEdit">
      <property name="placeholderText">
       <string>Entrez votre message ici</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sendButton">
      <property name="text">
       <string>Envoyer Message</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="messagesTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="commandLineEdit">
      <property name="placeholderText">
       <string>Entrez votre commande de mouvement ici</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="moveButton">
      <property name="text">
       <string>Envoyer Commande de Mouvement</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="commandsTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fetchDataButton">
      <property name="text">
       <string>Récupérer Données</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="captureScreenButton">
      <property name="text">
       <string>Capturer Écran</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="measurementsTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="commandLabel">
      <property name="text">
       <string>Q - Tourner à gauche, X - Arrêter moteurs, D - Tourner à droite, S - Reculer, A - Tours, W - Recule à gauche, C - Recule à droite</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="moveUpButton">
      <property name="text">
       <string>Haut</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="moveDownButton">
      <property name="text">
       <string>Bas</string>
      </property>
     </widget>
    </item>

    <!-- Ajout contrôle du servomoteur -->
    <item>
     <widget class="QSlider" name="servoSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="value">
       <number>90</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sendServoButton">
      <property name="text">
       <string>Envoyer angle au servomoteur</string>
      </property>
     </widget>
    </item>
    <!-- Fin ajout contrôle servomoteur -->

   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
